src/pages/qna/QnA.tsx

```
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import inferenceExamples from "./qna.ts";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)